app/src/main/java/project/com/inflix_android/ui/fragment/LoginFragment.kt

```diff
@@ -31,4 +29,29 @@ class LoginFragment : Fragment() {
     ): View? {
         return inflater.inflate(R.layout.fragment_login, container, false)
     }
+
+    override fun onActivityCreated(savedInstanceState: Bundle?) {
+        super.onActivityCreated(savedInstanceState)
+
+        lateinit var loginPresenter : LoginPresenterInterface
+
+        //Init
+        loginPresenter = LoginPresenter(this)
+        //Event
+        val button : Button? = view?.findViewById<Button>(R.id.login_button)
+        button!!.setOnClickListener {
```

Comment:
```suggestion
        button?.setOnClickListener {
```
Não é uma boa prática utilizar o operador `!!` dessa forma.